napari_tiff/napari_tiff_reader.py

```diff
@@ -72,14 +70,30 @@ def zip_reader(path: PathLike) -> List[LayerData]:
 
 
 def tifffile_reader(tif):
-    """Return napari LayerData from largest image series in TIFF file."""
+    """Return napari LayerData from image series in TIFF file."""
+    nlevels = len(tif.series[0])
+    if nlevels > 1:
+        data = [da.from_zarr(tif.aszarr(level=level)) for level in range(nlevels)]
```

Comment:
This assumes that the tiff series are arranged in a pyramid format. Is that something that is absolutely guaranteed by the tiff specification? 
If not, we may need to check we actually have a multi-resolution file with all the resolution levels in the right order before passing it to napari.

Reply:
I've not seen this written otherwise in any WSI file format. I imagine it's spec.

Reply:
Doctring from series `Series of pages with compatible shape and data type.` So if the length of series is different from 1 then different shapes are present. 

However, I'm not sure if it always should be read as pyramidal. As I know that tiff allows storing thumbnail for example. But I do not have such data available.

Reply:
@Czaki this is testing the length / number of levels inside the first serie. Would a thumbnail be the same or a separate series? What would the best way be to test for pyramid - ome metadata?

Reply:
I do not know. I work with writing my own tiff parser a long time ago and cannot find the data that I used then. This is rather point that it will be nice to check this (for example using some public datasets). I will also try to digg it later.